### ISSUE TITLE
Add setting for one import per line

### ISSUE THREAD

**Issue by clayne11** [buehler/typescript-hero#351]
#### Feature: One import per line
When adding / organizing imports, I like to only have one import per line. Add a setting to decide if all the imports should be on one line of multiple lines.

#### Reason
It makes life a lot easier for certain changes that need to be made by doing a global find and replace.

**Comment by buehler** [comment]
So you actually do import each alias from a lib with one line?
e.g.:
```typescript
import { foo } from 'lib';
import { bar } from 'lib';
```

instead of `{foo,bar} from 'lib'` ?

**Comment by clayne11** [comment]
Yup. We enforce that in our codebase and I can't count the number of times it's helped with refactoring code. The refactoring tooling in TS is nice but not 100% reliable IME. Sometimes refactoring can also move a file from one module to another which would be pretty difficult to handle with code intelligence tools but is super easy to do with a global find and replace.

**Comment by lnunno** [comment]
I'd prefer something like:

```
import {
  foo,
  bar,
} from 'lib';
```

I haven't found an option in vscode that supports this preference via formatter.
